test/integration/default/serverspec/app_spec.rb

```diff
@@ -13,22 +13,22 @@
   end
 
   it 'still serves Chef Supermarket when Unicorn is restarted' do
-    restart = command 'supermarket-ctl restart rails; sleep 5'
+    restart = command 'supermarket-ctl restart rails; sleep 5' # rubocop: disable Lint/UselessAssignment
     cmd = command 'wget --no-check-certificate -O - https://localhost 2> /dev/null'
```

Comment:
What do you ya'll think about—and maybe for a future PR—checking for some standard content in the output of the response to `GET https://localhost` so that we're not succeeding if nginx says "Hi! I'm here!"?